src/test/java/seedu/dengue/testutil/PersonListBuilder.java

```diff
@@ -0,0 +1,152 @@
+package seedu.dengue.testutil;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import seedu.dengue.model.person.Person;
+
+/**
+ * A utility class to help with building Lists of Person objects.
+ */
+public class PersonListBuilder {
```

Comment:
I was following what they did for PersonBuilder :P I thought it was useful because I use the same functionality for 6 classes and didn't want to repeat chunks of List-building code. It may be more LoC, but it's far neater, I find.

What do you think?

Reply:
I find this acceptable too!

Reply:
Since it's used multiple times actually in the case of analyst classes, and we have a moving personList, I think this is justified